artifacts.go

```diff
@@ -56,14 +59,42 @@ func artifactsHandler(packagesBasePaths []string, cacheTime time.Duration) func(
 		w.Header().Set("Content-Type", "application/gzip")
 		cacheHeaders(w, cacheTime)
 
-		err = archiver.ArchivePackage(w, archiver.PackageProperties{
+		properties := archiver.PackageProperties{
 			Name:    packageName,
 			Version: packageVersion,
 			Path:    packagePath,
-		})
-		if err != nil {
-			log.Printf("archiving package path '%s' failed: %v", packagePath, err)
+		}
+		// TODO: find proper directory, make it a config option
```

Comment:
I think it needs to be a more advanced concept. We should define an upper limit of total cache size and remove based on LRU rules. Otherwise, we can easily loose space there.